### PR TITLE
Call cp2k in the CI workflows

### PR DESCRIPTION
**Pull request by felipeZ**
Install and Run CP2K in the CI workflows

### CODE REVIEW

File: src/qmflows/test_utils.py [hunk 1]
```diff
@@ -37,6 +41,47 @@
 #: The path to the ``tests/test_files/molecules`` directory.
 PATH_MOLECULES = PATH / "molecules"
 
+#: Basisset specification for CP2K calculations
+kinds_template = yaml2Settings("""
```
Comment (BvB93) [inline src/qmflows/test_utils.py]: Yay, more test utilities!

File: src/qmflows/test_utils.py [hunk 2]
```diff
@@ -26,6 +28,8 @@
 from pathlib import Path
 from typing import Any, Callable, Union
 
+from qmflows.fileFunctions import yaml2Settings
```
Comment (BvB93) [inline src/qmflows/test_utils.py]: I'd recommend using relative import paths.

File: test/test_cp2k_mock.py [hunk 3]
```diff
@@ -6,54 +6,16 @@
...
 from qmflows.packages.cp2k_package import CP2K_Result
-from qmflows.test_utils import PATH, PATH_MOLECULES
+from qmflows.test_utils import PATH, PATH_MOLECULES, fill_cp2k_defaults
 from qmflows.utils import init_restart
 
 # module constants
 WORKDIR = PATH / "output_cp2k"
 ETHYLENE = Molecule(PATH_MOLECULES / "ethylene.xyz")
 
-kinds_template = yaml2Settings("""
```
Comment (BvB93) [inline test/test_cp2k_mock.py]: Nice, lots of cleaning house here.

File: src/qmflows/test_utils.py (hunk 2 above)
Comment (felipeZ) [reply src/qmflows/test_utils.py]: thanks!

**Pull request merged**